partner_identification/__manifest__.py

```diff
@@ -9,7 +9,7 @@
 {
     "name": "Partner Identification Numbers",
     "category": "Customer Relationship Management",
-    "version": "13.0.1.0.1",
+    "version": "13.0.1.0.2",
```

Comment:
You should not bump versions yourself, to avoid unnecessary merge conflicts. The OdooBot does that automatically.

Reply:
I see you made XML changes. So the "minor" number shoudl be bumped instead (-> 13.0.1.*1*.0), signaling that a module upgrade is needed.